.github/workflows/static-analysis.yml

```diff
@@ -16,6 +16,12 @@ jobs:
     steps:
       - name: "Checkout"
         uses: "actions/checkout@v3"
+      - name: PHPStan
+        uses: "docker://oskarstark/phpstan-ga"
+        env:
+          REQUIRE_DEV: true
+        with:
+          args: analyze --no-progress
```

Comment:
this makes it part of the coding-standards job.

if we define it as separate job, it runs in parallel with the cs fixer and also fails separately so its immediatly clear which things are good and which not. same as in https://github.com/php-http/httplug/blob/2.x/.github/workflows/static.yml (we could rename the `coding-standards` job to `php-cs-fixer` to make it fully clear which is which.

Reply:
Totally agree!